.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes _URL of issue_.
```

Comment:
```suggestion
Fixes *ISSUE_URL*
```

I find that all caps makes it more obvious when something is a placeholder that need replacing. And putting an underscore instead of a space between words makes it easier to select by double-clicking (https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/).

Also, not sure if it's necessary, but replaced markdown underscores with asterisks (to avoid potential issues with underscore in text) and removed full-stop to avoid it potentially becoming part of the pasted URL.